lib/archethic_web/live/last_ten_transactions.ex

```diff
@@ -0,0 +1,157 @@
+defmodule ArchethicWeb.LastTenTransactionsComponent do
```

Comment:
I think the name of module could be more generic like `TopTransactionsComponent`